node_modules/oae-principals/lib/init.js

```diff
@@ -71,7 +71,14 @@ var _ensureSchema = function(callback) {
         'PrincipalsEmailToken': 'CREATE TABLE "PrincipalsEmailToken" ("principalId" text PRIMARY KEY, "email" text, "token" text)',
 
         // Track user visits to groups they are members of
-        'UsersGroupVisits': 'CREATE TABLE "UsersGroupVisits" ("userId" text, "groupId" text, "latestVisit" text, PRIMARY KEY ("userId", "groupId"))'
+        'UsersGroupVisits': 'CREATE TABLE "UsersGroupVisits" ("userId" text, "groupId" text, "latestVisit" text, PRIMARY KEY ("userId", "groupId"))',
+
+        // 
+        'ArchiveByTenant': 'CREATE TABLE "ArchiveByTenant" ("aliasTenant" text PRIMARY KEY, "idArchive" text)',
+
+        // 
+        'DataArchive': 'CREATE TABLE "DataArchive" ("idArchive" text, "principalId" text, "idResources" text, PRIMARY KEY ("idArchive", "principalId", "idResources"))'
```

Comment:
As above with `archiveId` and `resourceId`. I'd also prefer having a row per resource instead of storing all the resource IDs in an array.